relay-event-schema/src/processor/funcs.rs

```diff
@@ -0,0 +1,54 @@
+use relay_protocol::{Annotated, IntoValue, Meta};
+
+use crate::processor::{
+    ProcessValue, ProcessingAction, ProcessingResult, ProcessingState, Processor,
+};
+
+/// Modifies this value based on the action returned by `f`.
+#[inline]
+pub fn apply<T, F, R>(v: &mut Annotated<T>, f: F) -> ProcessingResult
```

Comment:
This used to be a method on `Annotated`. Since the signature requires types from processors that aren't available to `relay-protocol`, the method had to be moved to a utility function in the schema.

Originally, `ProcessingAction` was supposed to be a simple enum declaring whether or not a value should be removed as part of applying a transform function on it. At some point, the `InvalidTransaction` variant got added to it, probably mistaking this enum for an error. That variant has since stuck, and more logic has built on that new model. It's now too much effort to change this back.